usaspending_api/disaster/tests/integration/test_disaster_agency_spending.py

```diff
@@ -97,17 +103,16 @@ def test_basic_success(client, disaster_account_data, elasticsearch_award_index,
             "code": "007",
             "description": "Agency 007",
             "children": [],
-            "award_count": 1,
+            "award_count": 2,
```

Comment:
This bothers me, I didn't see a new award added to the mock data but I could have missed it. Was the original API or test wrong?

Reply:
This is due to changing from using `award_id` to the `financial_account_distinct_award_key`